src/js/stores/ui.js

```diff
@@ -146,6 +146,15 @@ define(function (require, exports, module) {
          */
         components: UI_COMPONENTS,
 
+        /**
+         * Reference point for a layer when adjusting W and H
+         * Default is "lt" for left, top
```

Comment:
Thanks for this catch. For the record, "lt" is correct and I changed all the "tl"'s.